src/Utils/CliMsgFormatter.php

```diff
@@ -95,7 +95,26 @@ public function yellow( string $text ): string {
 	 * @return string
 	 */
 	public function progress( int $i, int $total ): string {
-		return min( 100, round( ( $i / $total ) * 100 ) ) . " %";
+		// @codeCoverageIgnoreStart
+		return $this->calculateProgress( $i, $total, true );
+		// @codeCoverageIgnoreEnd
+	}
+
+	/**
+	 * @codeCoverageIgnore
+	 */
```

Comment:
What is the deal with those ignore tags? I don't see why this code should be ignored. If you added them because the coverage bot gets mad otherwise, we should change the coverage bot

Reply:
We can't test this function as it's private.